tasks/update-cnb-dependency/buildpack_toml.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/BurntSushi/toml"
+	"github.com/pkg/errors"
+)
+
+type BuildpackToml struct {
+	API       string
+	Buildpack struct {
+		ID      string
+		Name    string
+		Version string
+	} `toml:"buildpack"`
+	Metadata Metadata
+	Order    []Order
+	Stacks   []Stack
+}
+
+type Metadata struct {
+	IncludeFiles               []string `toml:"include_files"`
+	Dependencies               []Dependency
+	DependencyDeprecationDates []DependencyDeprecationDate `toml:"dependency_deprecation_dates"`
+}
+
+type Stack struct {
+	ID string
+}
+
+func (buildpackToml BuildpackToml) LoadExpandedDependencies() []Dependency {
+	return expandDependenciesForEachStack(buildpackToml.Metadata.Dependencies)
```

Comment:
I think this can be a method on `Dependencies`, which would make the organization a touch clearer.